Dispatcher/Controller/LaunchScreenViewController.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+class LaunchScreenViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        
+        Timer.scheduledTimer(withTimeInterval: 3.0, repeats: false) { timer in
```

Comment:
we do not want to use a Timer for a simple task as waiting 3 seconds
use asyncAfter